.github/workflows/pr-ci.yml

```diff
@@ -274,6 +274,64 @@ jobs:
           name: built-projects
           path: dist/*
         if: env.SHOULD_RUN == 'true'
+  #
```

Comment:
Curious why this is a whole separate action vs just another step in the build workflow?

Reply:
My initial thought was that this step depends on build steps to be successful. But you might be right, if it's listed after build in one step, it has the same effect, right?